docs/3.x.x/en/advanced/customize-admin.md

```diff
@@ -99,7 +101,7 @@ Note: make sure the size of your image is the same as the existing one (434px x
 
 ## Build
 
-To build the administration, run the following command from the `./admin` folder:
+To build the administration, run the following command from the root directory of your project, **not** the `./admin folder:
```

Comment:
Running `setup` on the root directory executes a `cd admin && npm run setup` ... is running `npm run setup` in `/admin`. I don't see any difference.

Reply:
*shrug* I'm only fixing this for @soupette.

Reply:
I also agree but in all the GitHub issues I always say to run this command at the root of your strapi project (old habits die hard) therefore I think that the documentation should be similar as the answer given on GitHub

Reply:
Yeah, but the this "note" on the documentation has a tone of mandatory and that's false; right now, the command should works either on the root directory or `/admin`. IMHO this note add just a "bad feeling" sensation.

Reply:
@maturano In my case during testing (among others on the slack) running this command within the /admin directory failed.

Thus why this PR was suggested.